public/project/css/landing.css

```diff
@@ -0,0 +1,61 @@
+/* Landing Container */
+.landing-container {
+  background: url('http://www.wall321.com/thumbnails/detail/20120308/cityscapes%20city%20cars%20aston%20martin%20james%20bond%20bridges%20silver%20cars%20dbs%201920x1200%20wallpaper_www.wall321.com_15.jpg') no-repeat center center fixed;
+  background-size: cover !important;
+  background-position: center;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  box-shadow: 0 0 85px black;
+}
+
+/* Landing Overlay */
+.landing-overlay {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: rgba(1, 1, 1, 0.7);
+}
+
+/* Landing Center Content Container */
+.landing-content {
+  width: 400px;
+  height: 200px;
+  max-width: 100%;
+}
+
+/* Landing Title */
+.landing-title {
+  position: relative;
+  color: white;
+  font-size: 48px;
+  font-weight: 300;
+  text-align: center;
+  margin-top: 14px;
+  white-space: nowrap;
+}
```

Comment:
Since landing-title is the first element in the container, it is better to set a padding-top on the parent container instead of applying margin-top. Margin is something that is hectic when it comes to working cross browsers, especially pc. So instead of doing margin-top:14px, set padding-top:14px to the parent container. Also, you do not need white-space: nowrap. You just need to set the appropriate min-width to landing-content.